clang/include/clang/Basic/Attr.td

```diff
@@ -1997,9 +1997,12 @@ def Convergent : InheritableAttr {
 def NoInline : DeclOrStmtAttr {
   let Spellings = [CustomKeyword<"__noinline__">, GCC<"noinline">,
                    CXX11<"clang", "noinline">, C23<"clang", "noinline">,
+                   CXX11<"msvc", "noinline">, C23<"msvc", "noinline">,
                    Declspec<"noinline">];
   let Accessors = [Accessor<"isClangNoInline", [CXX11<"clang", "noinline">,
```

Comment:
We should probably have a different name for this, since it is no longer accurate.  Perhaps `isStmtNoInline`

Reply:
okay, got it.

Reply:
`[[msvc::noinline]]` is just an alias of `[[clang::noinline]]`, and maybe we don't really need UT for it, so I just deleted the redundant test files. What do you think?

Reply:
done